EncryptionBoxInput/README.md

```diff
@@ -0,0 +1,76 @@
+
+# EncryptionBoxInput Interface
+
+**Status**: Proposed
+
+| Name               | ID                                                                |
+| :----------------- | :---------------------------------------------------------------- |
+| EncryptionBoxInput | 5b5f76b54d976d72f1ada3063d1af2e5352edaf1ba86b3b311170d4d81056d61  |
+
+
+## Description
+
+Interface for creating new encryption box that can encrypt/decrypt data using defined crypto algorithm.
+
+
+## Functions
+
+`get` - prints prompt message to the user and returns handle to opened encryption box.
+
+arguments:
+
+    answerId: uint32 - id of function callback.
+    prompt: bytes - utf-8 string to print to the user before input, OPTIONAL (empty string is similar to null).
+    algorithm: bytes - utf-8 string with a name of encryption algorithm. Can be one of the following value: "NaclBox",  "NaclSecretBox", "Chacha20" - or other algorithm name supported by Browser.
+    params: TvmCell - packed algorithm parameters.
```

Comment:
as @mikhailUshakoff noted we are to describe the structure of the params here

Reply:
Deprecated